pvacseq/lib/call_iedb.py

```diff
@@ -49,6 +49,8 @@ def main(args_input = sys.argv[1:]):
 
     if args.iedb_executable_path is not None:
         response = run(prediction_class_object.iedb_executable_params(args), stdout=PIPE)
+        if response.returncode == 2:
```

Comment:
This is a result of IEDB using `OptionParser`. It returns code 2 when `parser.error` is set. See https://docs.python.org/2/library/optparse.html#how-optparse-handles-errors and the `main` function in /gscuser/ssiebert/mhc_i/src/predict_binding.py.